lib/cache/cache_test.go

```diff
@@ -2038,6 +2062,72 @@ func TestDatabases(t *testing.T) {
 	require.Equal(t, 0, len(out))
 }
 
+func TestRelativeExpiry(t *testing.T) {
+	const checkInterval = time.Second
+	const nodeCount = int64(100)
+
+	ctx := context.Background()
+
+	clock := clockwork.NewFakeClockAt(time.Now().Add(time.Hour))
+	p := newTestPack(t, func(c Config) Config {
+		c.RelativeExpiryCheckInterval = checkInterval
+		c.Clock = clock
+		return ForAuth(c)
+	})
+	t.Cleanup(p.Close)
+
+	// add servers that expire at a range of times
+	now := clock.Now()
+	for i := int64(0); i < nodeCount; i++ {
+		exp := now.Add(time.Minute * time.Duration(i))
+		server := suite.NewServer(types.KindNode, uuid.New().String(), "127.0.0.1:2022", apidefaults.Namespace)
```

Comment:
This doesn't compile, since this file is importing `pborman/uuid` instead of `google/uuid`.

(With `pborman`, `uuid.New()` returns a string.